### ISSUE TITLE
UnknownError exception name mismatch

### ISSUE THREAD

**Issue by kgutwin** [ianunruh/hvac#16]
After my token expires:
```
>>> client.is_authenticated()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/hvac/v1/__init__.py", line 270, in is_authenticated
    self.lookup_token()
  File "/usr/lib/python2.7/site-packages/hvac/v1/__init__.py", line 228, in lookup_token
    return self._get('/v1/auth/token/lookup-self').json()
  File "/usr/lib/python2.7/site-packages/hvac/v1/__init__.py", line 357, in _get
    return self.__request('get', url, **kwargs)
  File "/usr/lib/python2.7/site-packages/hvac/v1/__init__.py", line 400, in __request
    raise exceptions.UnknownError()
AttributeError: 'module' object has no attribute 'UnknownError'
```
It looks like `hvac.exceptions` doesn't have an UnknownError object, but does have an UnexpectedError object.

**Comment by ianunruh** [comment]
Yep, that's what it should be throwing. Out of curiosity though, what HTTP error is being returned from Vault? That if/else should cover all documented HTTP errors from Vault.

**Comment by kgutwin** [comment]
Response code 403, errors is `[u'bad token']`.

**Comment by kgutwin** [comment]
FWIW, the version of Vault is very recent, as of yesterday (0.2.0 prerelease)

**Comment by ianunruh** [comment]
Thanks! As of today, https://github.com/hashicorp/vault/blob/master/website/source/docs/http/index.html.md#http-status-codes does not document the use of `403 Forbidden`. This should probably be raised on their issue tracker.

**Comment by kgutwin** [comment]
The change appeared in hashicorp/vault#366. I'll mention it there.

**Comment by ianunruh** [comment]
Fixed in 27b5b2915df90a2c56fda2f87beed2399308b21e
